spyder/plugins/editor/widgets/editor.py

```diff
@@ -1568,6 +1571,20 @@ def close_all_but_this(self):
         for i in range(0, self.get_stack_count()-1  ):
             self.close_file(0)
 
+    """ Sort open tabs alphabetically """

```

Comment:
Please move this docstring to be inside the function definition (just below `def sort_file_tabs(self)`). Also remove the spaces around the text and add a dot at the end.

Reply:
Made change